src/test/java/org/sonar/plugins/stash/issue/StashDiffReportTest.java

```diff
@@ -76,7 +89,7 @@ public void testAddReport() {
 
   @Test
   public void testGetType() {
-    assertEquals(IssueType.CONTEXT, report1.getType("path/to/diff1", 20, StashDiffReport.VICINITY_RANGE_NONE));
+    assertEquals(null, report1.getType("path/to/diff1", 20, StashDiffReport.VICINITY_RANGE_NONE));

```

Comment:
AFAIU, `vicinity=0` means we only care about issue in changed code. So it should be impossible for `StashDiffReport.getType()` to return `CONTEXT` if vicinity was 0.

Reply:
True. Can you use assertNull() for the null assertions?

Reply:
Done 👍